### DISCUSSION TITLE
Seperate providers for RAG and LLM?

### DISCUSSION THREAD

**Discussion by zmccj** [pfrankov/obsidian-local-gpt#39] [General]
How nice a plugin, clean and handy, I use it a lot.
For consideration of data privacy and cost of token, is this possible to have ollama for text embedding, and openAI for llm response?
I'm new to LLM, so maybe this is a silly question.
Thanks a lot for developing this wonderful plugin~

**Comment by pfrankov** [comment]
Thank you for the very warm words!
Yes, this feature is in the plan.
